helm/idol-nifi/templates/nifi/nifi.yaml

```diff
@@ -53,6 +54,37 @@ spec:
         securityContext: {{- omit $component.containerSecurityContext "enabled" | toYaml | nindent 10 }}
           readOnlyRootFilesystem: true
         {{- end }}
+      - name: ensure-conf
+        image: {{ include "idol-library.idolImage" (dict "root" $root "idolImage" $component.idolImage) }}
+        envFrom:
+        - configMapRef:
+            name: idol-nifi-env
+            optional: false
+        resources:
+          requests:
+            cpu: {{ $component.nifi.resources.requests.cpu }}
+            memory: {{ printf "%dMi" ( $component.nifi.resources.requests.memoryMi | int) }}
+          limits:
+            cpu: {{ $component.nifi.resources.limits.cpu }}
+            memory: {{ printf "%dMi" ( $component.nifi.resources.limits.memoryMi | int) }}
```

Comment:
Don't really need to apply the same cpu/memory limits for the init container if we don't execute the nifi main